prov/netdir/src/netdir_init.c

```diff
@@ -141,6 +141,7 @@ static int ofi_nd_adapter_cb(const ND2_ADAPTER_INFO *adapter, const char *name)
 	info->fabric_attr->prov_version = OFI_VERSION_DEF_PROV;
 
 	info->caps = OFI_ND_EP_CAPS | OFI_ND_DOMAIN_CAPS;
+	info->mode = FI_CONTEXT;
```

Comment:
Should info->tx_attr->mode and info->rx_attr_mode also be set to FI_CONTEXT?

If I understood what I read in the documentation correctly, I gathered that it would be possible for a provider to, for example, require context for fi_sendv, but not fi_recvv. In that scenario it seemed like one should expect that FI_CONTEXT would be set for info->mode and info->tx_attr->mode, but cleared for info->rx_attr->mode.

Reply:
Yes, those should be set as well.  I'll update.